assets/js/displayMemberNames.js

```diff
@@ -0,0 +1,34 @@
+import $ from "jquery";
+
+function changeDisplayNames() {
+  const checkBox = document.getElementById("name_display_toggle");
+  const teamNamesCurrentlyDisplayed = !checkBox.checked;
+
+  if (teamNamesCurrentlyDisplayed) {
+    const teamNames = document.getElementsByClassName("teamNames");
+    for (let i = 0; i < teamNames.length; i += 1) {
+      const element = teamNames[i];
+      element.style.display = "block";
+    }
+    const memberNames = document.getElementsByClassName("memberNames");
+    for (let i = 0; i < memberNames.length; i += 1) {
+      const element = memberNames[i];
+      element.style.display = "none";
+    }
+  } else {
+    const teamNames = document.getElementsByClassName("teamNames");
+    for (let i = 0; i < teamNames.length; i += 1) {
+      const element = teamNames[i];
+      element.style.display = "none";
+    }
+    const memberNames = document.getElementsByClassName("memberNames");
+    for (let i = 0; i < memberNames.length; i += 1) {
+      const element = memberNames[i];
+      element.style.display = "block";
+    }
+  }
```

Comment:
I'm not up to spec on best practices for JS, but this worked for me locally. Avoids the looping but no clue if its technically idiomatic.

```suggestion
  let styleSheet = document.getElementById("nameStyling");
  
  styleSheet.innerHTML = teamNamesCurrentlyDisplayed
      ? `.teamNames { display: block !important; } .memberNames { display: none !important; }`
      : `.teamNames { display: none !important; } .memberNames { display: block !important; }`;
```

Then you add this to pairing_display.html and pretty_pairing.html
`<style id="nameStyling" type="text/css">
    </style>`